practice/src/index.html

```diff
@@ -38,5 +38,69 @@
       </div>
     </div>
   </header>
+
+  <main class="main-section">
+    <!-- Best Place section -->
+    <section class="container place-section">
+      <div class="wrapper-place">
+        <img src="../src/assets/images/banner.webp" alt="" class="image-banner">
+        <hgroup class="place-introduction">
+          <h1 class="text place-title">Find The Best Place <br />to Stress Release</h1>
+          <p class="text place-description">A new way to relieve your stress by going on vacation!<br />Find the trip you want the
+            book it immediately!</p>
+        </hgroup>
+        <div class="tab-nav">    
+          <ul class="tab-list">
+            <li class="tab-item native">
+              <a href="" class="tab-link active">Hiking</a>
+            </li>
+            <li class="tab-item">
+              <a href="" class="tab-link">Travel</a>
+            </li>
+          </ul>
+          <div class="tab-pane">
+            <div class="pane-1">
+              <div class="pane-content">
+                <img src="../src/assets/images/icon-address.svg" alt="" class="icon-address">
+                <hgroup class="tab-introduction">
```

Comment:
The `<hgroup>` groups a single `<h1>–<h6>` element with one or more `[<p>]`. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup